riscv/src/riscv_asm.lalrpop

```diff
@@ -189,6 +189,6 @@ Symbol: String = {
 }
 
 Number: i64 = {
-    r"-?[0-9][0-9_]*" => i64::from_str(<>).unwrap().into(),
-    r"0x[0-9A-Fa-f][0-9A-Fa-f_]*" => i64::from_str_radix(&<>[2..].replace('_', ""), 16).unwrap().into(),
+    r"-?[0-9][0-9_]*" => i64::from_str(<>).unwrap(),
+    r"0x[0-9A-Fa-f][0-9A-Fa-f_]*" => u64::from_str_radix(&<>[2..].replace('_', ""), 16).unwrap() as i64,
```

Comment:
I still don't understand this, why not parse as i64 right away? As it is now, we parse a big 64 bit number and make it negative without noticing. Does this happen?

Reply:
No, currently `unwrap()` panics if number is > 2**63.

Reply:
(what makes sense, as I am parsing i64 and the number is beyond max)